README.md

```diff
@@ -170,4 +170,4 @@ More ready-to-use examples can be found in the [Example](https://mosecorg.github
 
 ## Contributing
 
-We welcome any kind of contribution. Please give us feedback by [raising issues](https://github.com/mosecorg/mosec/issues/new/choose) or directly [contribute](https://mosecorg.github.io/mosec/contributing) your code and pull request!
+We welcome any kind of contribution. Please give us feedback by [raising issues](https://github.com/mosecorg/mosec/issues/new/choose) or discussing on [Discord](https://discord.gg/DhQqWn8e). You could also directly [contribute](https://mosecorg.github.io/mosec/contributing) your code and pull request!
```

Comment:
Can also add to the badge.

Ref

- https://shields.io/
- https://raw.githubusercontent.com/tensorchord/envd/main/README.md